src/wagtail_ai/wagtail_hooks.py

```diff
@@ -6,20 +6,22 @@
 from django.urls import include, path, reverse
 from django.utils.html import format_html, json_script
 from django.views.i18n import JavaScriptCatalog
+from django_ai_core.contrib.agents import registry
 from wagtail import hooks
 from wagtail.admin.rich_text.editors.draftail.features import ControlFeature
 from wagtail.admin.staticfiles import versioned_static
 from wagtail.contrib.settings.models import register_setting
 
 from wagtail_ai.agents.base import get_agent_settings_model
 
-from .agents.content_feedback import ContentFeedbackAgent
 from .models import Prompt
 from .views import describe_image, prompt_viewset, text_completion
 
 
 @hooks.register("register_admin_urls")  # type: ignore
 def register_admin_urls():
+    content_feedback_agent = registry.get("wai_content_feedback")
```

Comment:
This allows

```py
from django_ai_core.contrib.agents import registry
from wagtail_ai.agents.content_feedback import ContentFeedbackAgent

class MyCustomContentFeedbackAgent(ContentFeedbackAgent):
    provider_alias = "scaleway"

    # Can also override methods to customize behavior


registry.register()(MyCustomContentFeedbackAgent)
```

in application code (e.g. `bakerydemo/base/wagtail_hooks.py`)

Also, now that I think about it, perhaps wagtail-ai should have its own registry instance, so we don't have to prefix the slugs with `wai_`. Then, we expose `registry` from `wagtail_ai.agents` instead of `from django_ai_core.contrib.agents import registry`.